cluster-scope/base/core/namespaces/examples-openshift-pub/namespace.yaml

```diff
@@ -4,3 +4,6 @@ metadata:
     name: examples-openshift-pub
     annotations:
         openshift.io/requester: rbo
+        op1st/project-owner: examples-openshift-pub
```

Comment:
This is not a valid Openshift group in [this list](https://github.com/operate-first/apps/tree/master/cluster-scope/base/user.openshift.io/groups).
@rbo Is there a group that should have access/ownership to this project?